config/crd/bases/operator.authorino.kuadrant.net_authorinoes.yaml

```diff
@@ -0,0 +1,133 @@
+
```

Comment:
authorinoEs?

Reply:
We should change to "authorinos" IMO. https://github.com/Kuadrant/authorino-operator/pull/1#discussion_r741065960

Reply:
@eguzki this name structure is generated by the operator-sdk, do you know if we can customize it?

Reply:
`operator-sdk create api` has this `--plural`. Did you tried it?

Reply:
I got it working by editing the `//+kubebuilder:resource:path="authorinos"` to `authorino_types.go` https://book.kubebuilder.io/reference/markers/crd.html?highlight=crd,plural#crd-generation